index.js

```diff
@@ -209,7 +210,7 @@ function mountfs_path(devPath, path, type, flags, extras, callback)
   catch(error)
   {
     // catch everything, but not "Error no Entry"
-    if(err.code !== 'ENOENT') return callback(error)
+    if(error.code !== 'ENOENT') return callback(error)
 
     // mount the filesystem
     if(devPath)
```

Comment:
@piranna i dont know if this is the intended behavior but should the function know if the devPath is actually a exisiting path?
Should we wrap this in a try/catch wit fs.statSync?